app/routers/dayview.py

```diff
@@ -105,6 +105,20 @@ def _check_multiday_event(self) -> Tuple[bool]:
 
 
 def event_in_day(event: Event, day: datetime, day_end: datetime) -> bool:
+    if event.all_day:
+        return None
+    return (
+        (event.start >= day and event.start < day_end) or
+        (event.end >= day and event.end < day_end) or
+        (event.start < day_end < event.end)
+        )
+
+
+def all_day_event_in_day(
+    event: Event, day: datetime, day_end: datetime
+) -> bool:
+    if not event.all_day:
+        return None
```

Comment:
Maybe `False`?

Reply:
What do you mean?

Reply:
This function returns boolean and it's important to keep up the consistency.